src/main.py

```diff
@@ -153,6 +157,16 @@ async def enquary_command(update: Update, context: ContextTypes.DEFAULT_TYPE) ->
 async def dev_command(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
     await update.message.reply_text("This feature is under development")
 
+\
+# Function to record user feedback and store it in feedback.txt in feedback folder
+async def feedback_receive(update: Update, context: CallbackContext) -> None:  
+    user_message = update.message.text
+    with open("Feedback/feedback.txt","a") as feedback_file:
+        feedback_file.write(f"\n{str(datetime.now())}: " + user_message)
+        feedback_file.close()
+               
+    global feedback
+    feedback = False
```

Comment:
**Review: Enhancements Suggested**

1. **Cancellation Option:** Integrate a `/cancel` command or the ability to type "cancel" for users to exit feedback mode without submitting feedback, enhancing user control.

2. **Context Manager Usage:** No need to explicitly close file with `feedback_file.close()` when using a context manager. Embracing this practice ensures cleaner and more concise code.